I've merged your changes and resolved the conflict in `package.json` but the tests are failing.

``` shell
npm test

> grunt-faker@0.2.0 test /Volumes/mystuff/dotnetCarpenter/Projects/buildTools/grunt-faker
> grunt

Running "clean:tests" (clean) task
Cleaning tmp...OK

Running "jshint:all" (jshint) task
>> 11 files lint free.

Running "clean:tests" (clean) task

Running "faker:test_all" (faker) task
>> The definitions module from Faker.js is not avail in this task.

Running "nodeunit:tests" (nodeunit) task
Testing address_test.js............OK
Testing company_test.jsF....
>> names - suffixes
>> Message: suffixes is a array
>> Error: 'string' === 'object'
>> at Object.exports.names.suffixes (test/company_test.js:15:10)
>> at Object.exports.names.setUp (test/company_test.js:8:5)

Testing faker_test.js.OK
Testing helpers_test.jsFF....FFF
>> names - randomNumber
>> Message: randomNumber is a number
>> Error: 'string' === 'number'
>> at Object.exports.names.randomNumber (test/helpers_test.js:15:10)
>> at Object.exports.names.setUp (test/helpers_test.js:8:5)

>> names - randomizeNum
>> Message: randomize is a number
>> Error: 'string' === 'number'
>> at Object.exports.names.randomizeNum (test/helpers_test.js:23:10)
>> at Object.exports.names.setUp (test/helpers_test.js:8:5)

>> names - shuffle
>> Message: shuffle created
>> Error: 9 === 5
>> at Object.exports.names.shuffle (test/helpers_test.js:62:10)
>> at Object.exports.names.setUp (test/helpers_test.js:8:5)

>> names - shuffle
>> Message: shuffle is an array
>> Error: 'string' === 'object'
>> at Object.exports.names.shuffle (test/helpers_test.js:63:10)
>> at Object.exports.names.setUp (test/helpers_test.js:8:5)

>> names - createCard
>> Error: Expected 9 assertions, 0 ran
>> at processImmediate [as _immediateCallback] (timers.js:330:15)

>> names - createCard
>> TypeError: Cannot read property 'length' of undefined
>> at Object.exports.names.createCard (test/helpers_test.js:70:27)
>> at Object.exports.names.setUp (test/helpers_test.js:8:5)

>> names - userCard
>> Error: Expected 8 assertions, 0 ran
>> at processImmediate [as _immediateCallback] (timers.js:330:15)

>> names - userCard
>> TypeError: Cannot read property 'length' of undefined
>> at Object.exports.names.userCard (test/helpers_test.js:85:27)
>> at Object.exports.names.setUp (test/helpers_test.js:8:5)

Testing internet_test.js.....OK
Testing lorem_test.jsF....
>> names - words
>> Message: words is an array
>> Error: 'string' === 'object'
>> at Object.exports.names.words (test/lorem_test.js:15:10)
>> at Object.exports.names.setUp (test/lorem_test.js:8:5)

Testing name_test.js...OK
Testing phone_number_test.js..OK
Testing random_test.jsFF.....................
>> random - number
>> Message: number is a number
>> Error: 'string' === 'number'
>> at Object.exports.random.number (test/random_test.js:15:10)
>> at Object.exports.random.setUp (test/random_test.js:8:5)

>> random - array_element
>> Message: array_element is a number
>> Error: 'string' === 'number'
>> at Object.exports.random.array_element (test/random_test.js:23:10)
>> at Object.exports.random.setUp (test/random_test.js:8:5)

Warning: 12/130 assertions failed (59ms) Use --force to continue.

Aborted due to warnings.
npm ERR! Test failed.  See above for more details.
npm ERR! not ok code 0
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,5 +1,5 @@
 language: node_js
 node_js:
-  - "0.8"
+  - "0.10"
 before_install:
   - npm install -g grunt-cli

--- a/tasks/faker.js
+++ b/tasks/faker.js
@@ -30,12 +30,13 @@ module.exports = function(grunt) {
     var pattern = /^(.*)\{\{([^()]+?)(\((.+)\))?\}\}(.*)$/g,
     match, func, args;
     var argArray = [], surroundings = [];
+    var retValue;
 
     while (match = pattern.exec(value)) {
-      surroundings[0] = match[1] || null; // if nothing then set the value explicit to null to avoid empty string concatenations
+      surroundings[0] = match[1];
       func = match[2];
       args = match[4];
-      surroundings[1] = match[5] || null; // if nothing then set the value explicit to null to avoid empty string concatenations
+      surroundings[1] = match[5];
     }
 
     if (args !== undefined ){
@@ -51,9 +52,19 @@ module.exports = function(grunt) {
       }
     }
     // return value if no Faker method is detected
-    return func ?
-      (surroundings[0] + executeFunctionByName(func,argArray) + surroundings[1]) :
+    retValue = func ?
+      executeFunctionByName(func,argArray) :
       value;
+
+    if(surroundings[0]) {	// prefix
+    	retValue = surroundings[0] + retValue;
+    }
+
+    if(surroundings[1]) {	// postfix
+    	retValue += surroundings[1];
+    }
+
+    return retValue;
   }
 
   // Execute function as string

--- a/tasks/faker.js
+++ b/tasks/faker.js
@@ -32,10 +32,10 @@ module.exports = function(grunt) {
     var argArray = [], surroundings = [];
 
     while (match = pattern.exec(value)) {
-      surroundings[0] = match[1];
+      surroundings[0] = match[1] || null; // if nothing then set the value explicit to null to avoid empty string concatenations
       func = match[2];
       args = match[4];
-      surroundings[1] = match[5];
+      surroundings[1] = match[5] || null; // if nothing then set the value explicit to null to avoid empty string concatenations
     }
 
     if (args !== undefined ){

